spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -556,6 +556,9 @@ def build_trains(app_id, testing_type)
     def update_build_trains!(app_id, testing_type, data)
       raise "app_id is required" unless app_id
 
+      # The request fails if this key is present in the data
+      data.delete("dailySubmissionCountByPlatform")
+
```

Comment:
@KrauseFx as per #3768 I found that this was the cause of my issues. I found that spaceship submits a lot more data in the request than the iTC website does for distributing TestFlight builds, and having this particularly key caused me to get an "operation_failed" error. Removing this fixes the error.